compose/src/main/java/com/bottlerocketstudios/compose/map/TestGoogleMapState.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.compose.map
+
+import androidx.compose.runtime.mutableStateOf
+import com.bottlerocketstudios.mapsdemo.domain.models.UserFacingError
+import com.google.android.gms.maps.model.LatLng
+
+val googleMapScreenStateTest = GoogleMapScreenState(
```

Comment:
@Josh-Gebbeken nit: This can be `internal`

```suggestion
internal val googleMapScreenStateTest = GoogleMapScreenState(
```